CenterForOpenScience/osf.io
# [IE] Unable to view files in a VOL [OSF-6432]

## Purpose

Fix the problem that view only links doesn't work in view files on IE.


## QA notes:

before change:
1. create a project and add a file to that project
2. log off, create a view only link and use the view only link in IE to visit the file page of that project
3. click on the file in the file grid inside that project
You will not be able to view that file
after change:
1. create a project and add a file to that project
2. log off, create a view only link and use the view only link in IE to visit the file page of that project
3. click on the file in the file grid inside that project
You should be able to view the file as expected and it behavior the same as other browser.


## Side effects



Even tho this fix makes view only links work in IE, still don't know what cause the problem. It looks like the view only link is not passed correctly in request.referrer. There might be a more elegant solution to make the behavior consistent with other browser.
## Ticket



https://openscience.atlassian.net/browse/OSF-6432
